docs/developer-guide/etendo-copilot/available-tools/ddl-tool.md

```diff
@@ -0,0 +1,94 @@
+
+- **REGISTER_COLUMNS:** This mode is executed automatically after the ADD_COLUMN mode. It creates columns on Etendo System from the database with the previous created columns by calling a webhook that runs a Java file named RegisterColumns. This mode ensures that is the correct and unique table where add the columns.
+
+- **SYNC_TERMINOLOGY:** Cleans the terminology, removing "_" and adding spaces. It works by calling a webhook to execute a Java file named SyncTermsWebHook.
+
+- **REGISTER_WINDOW_AND_TAB:** Registers a window and a table to show the data in the application, also add a menu to provide an easy access. It first checks if there is a window already created with the same name. If necessary, the tool can force the creation with a parameter called ForceCreate. This mode uses the RegisterWindowAndTabWebHook java file.
+
+- **REGISTER_FIELDS:** Registers the fields of a tab in Etendo System to be recognized for it. It works by calling a webhook to run a Java file.
+
+## Usage Example 
+
+![ddltool.png](../../../assets/developer-guide/etendo-copilot/available-tools/DDLTool.png)
+
+**User**: "I want to create a table with the name 'Table Test' and prefix 'copdev'."
+
+**Copilot**: "Let's begin by registering the table in the system.
+Can you provide me with the following details:
+Classname (or I can generate one automatically for you).
+Description of the table (or I can generate one automatically).
+Help comment for the table (or I can generate one automatically)."
+
+**User**: "Just generate automatically"
+
+**Copilot**: "The table "Table_Test" has been created successfully in the database.
+Now, please provide the specific columns you want to add to the table, including:
+Column name
+Column data type
+If the column can be null
+If the column has a default value (and what it is, if applicable)
+For example:
+Column name: dogColor
+Column type: String
+Can be null: Yes
+Default value: None"
+
+**User**: "add the column 'Specific Data' with a text, can be null and has not default value"
+
+**Copilot**: "The table "Table Test" has been successfully created and configured in the system. Here's a summary of the steps performed:
+Registered the table "Table Test" in the system.
+Created the table "Table_Test" in the database.
+Added a column "Specific Data" with type "Text" to the table.
+Registered the columns of the table in the system.
+Synchronized the terminology.
+Created a window and header tab for "Table Test".
+Registered the fields for the window tab in the system.
+Synchronized the terminology again.
+Added descriptions and help comments to the columns without them.
+Synchronized the terminology again.
+It's strongly recommended to compile and restart Etendo to apply all changes effectively."
+
+## Results
+
+**First Response**
+
+![ddltool1.png](../../../assets/developer-guide/etendo-copilot/available-tools/DDLTool1.png)
```

Comment:
<img width="607" alt="image" src="https://github.com/etendosoftware/docs/assets/106354215/443db76e-ea72-4040-9648-877ed542fb43">
These images are not in the commit